plenum/server/replica.py

```diff
@@ -2170,11 +2175,11 @@ def _request_three_phase_msg(self, three_pc_key: Tuple[int, int],
         return True
 
     def _request_pre_prepare(self, three_pc_key: Tuple[int, int],
-                             recipients: List[str] = None,
                              stash_data: Optional[Tuple[int, int, int]] = None) -> bool:
         """
         Request preprepare
         """
+        recipients = self.primaryName
```

Comment:
Now we request PrePrepares from primary only. Why do we still request Prepares from all the replicas, not from non-primaries only?

Reply:
I think we will have problem in view change, because primary node will have its own prepare from last  view change iteration when it was not primary.